test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1185,10 +1189,10 @@ public void TensorFlowSentimentClassificationTest()
             // For explanation on how was the `sentiment_model` created 
             // c.f. https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
             string modelLocation = @"sentiment_model";
-            var pipelineModel = mlContext.Model.LoadTensorFlowModel(modelLocation).ScoreTensorFlowModel(new[] { "Prediction/Softmax" }, new[] { "Features" })
+            using var pipelineModel = mlContext.Model.LoadTensorFlowModel(modelLocation).ScoreTensorFlowModel(new[] { "Prediction/Softmax" }, new[] { "Features" })
```

Comment:
Similar changes may need to be made in the AutoML code, and its use in Model Builder. Otherwise it will continue to leak there.

Perhaps: In a follow-up PR: add a using or call dispose() for TF models and TF prediction engines.

/cc @LittleLittleCloud

Reply:
I have updated the docs and samples. I could not figure out easily where the corresponding fixes need to be bade in AutoML, Model Builder and CodeGen. @LittleLittleCloud, can you please help with that?

---
In reply to: [396764702](https://github.com/dotnet/machinelearning/pull/4939#discussion_r396764702) [](ancestors = 396764702)